src/Modules/UngradedModules.jl

```diff
@@ -1194,6 +1194,15 @@ function SubModuleOfFreeModule(F::FreeMod{L}, A::MatElem{L}) where {L}
   return subModule
 end
 
+# missing method
+function sparse_row(A::MatElem)
```

Comment:
This is called in the method just above this block. That's why I put it here for the time being.